### PR TITLE
Information on being a convener

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Write down some of the expectations we have for
working group conveners and some guidance on
how to run working groups.

### CODE REVIEW

File: what_are_WGs.md [hunk 1]
```diff
@@ -7,27 +7,73 @@ layout: default
...
 
 If you want to form a working group, or want to put some community-wide
 activity under the umbrella of the HSF,
 just contact the
 [HSF coordination team](mailto:hsf-coordination@googlegroups.com).
 
 Working Groups are added to the website of the HSF, with 
-description and contact information and will have a dedicated mailing list
-and Indico category.
+description and contact information and their own
```
Comment (eduardo-rodrigues) [inline what_are_WGs.md:27]:
```suggestion
description and contact information, and their own
```

File: what_are_WGs.md [hunk 2]
```diff
@@ -7,27 +7,73 @@ layout: default
...
+- be there to help encourage the exchange of ideas and techniques
+- do not need to focus on "finished" items, but rather can be there
+  to provide a community forum for unsolved problems
+- should ensure there is plenty of time for discussion
+- should take notes or minutes for important discussion points (attaching these
+  to Indico is recommended)
+
+We strongly encourage the use of common software in the HSF and opportunities
+for common development and sharing should be seized when they exist. The HSF
+exists as a [GitHub organisaion](https://github.com/HSF) and is happy
```
Comment (eduardo-rodrigues) [inline what_are_WGs.md:59]:
```suggestion
exists as a [GitHub organisation](https://github.com/HSF) and is happy
```

File: what_are_WGs.md [hunk 3]
```diff
@@ -7,27 +7,73 @@ layout: default
...
+to host projects to help them grow. This can include things like common
+datasets used in development as well as actual code.
+
+One focus that we encourage is to organise events together with other
+groups, e.g., a special session at a workshop, a joint meeting with
+a software development project, a cross-cutting topic that bridges
+more than one HSF working group (our working groups are not intended to 
+be silos). These meetings are excellent for bringing together
+people that might not normally talk or know each other.
```
Comment (eduardo-rodrigues) [inline what_are_WGs.md:68]:
```suggestion
people that might not normally talk to, or know, each other.
```

File: what_are_WGs.md [hunk 4]
```diff
@@ -7,27 +7,73 @@ layout: default
...
+
+Our working group conveners should usually dedicate between 5% and 10% of their
+time to running the working group. More than 10% is great, but we recognise
+that people are busy with many other tasks. Less than 5% and the working group
+can fall into an inactive state. Conveners should meet at least once a month to
+plan. The HSF Coordination Group are always happy to attend to give feedback
```
Comment (eduardo-rodrigues) [inline what_are_WGs.md:74]:
```suggestion
plan. The HSF Coordination Group is always happy to attend to give feedback
```

**Review by eduardo-rodrigues — APPROVED** [review]
I had a quick look and made 3-4 trivial suggestions. Overall ready. Thanks.

**Comment by roiser** [comment]
Do you want to point somewhere to the HSF workshops and say wg conveners could be asked to organise and convene sessions?

**Comment by eduardo-rodrigues** [comment]
> Do you want to point somewhere to the HSF workshops and say wg conveners could be asked to organise and convene sessions?

Good point.

**Comment by graeme-a-stewart** [comment]
> Do you want to point somewhere to the HSF workshops and say wg conveners could be asked to organise and convene sessions?

Yes, good point - I added something on this. Thanks.

**Comment by graeme-a-stewart** [comment]
> I had a quick look and made 3-4 trivial suggestions. Overall ready. Thanks.

Thanks a lot @eduardo-rodrigues !

**Comment by graeme-a-stewart** [comment]
OK, didn't get more comments, so merging. Thanks @roiser and @eduardo-rodrigues !

**Pull request merged**